lib/gdpr/use_case/obfusticate_sponsors.rb

```diff
@@ -0,0 +1,11 @@
+class Gdpr::UseCase::ObfusticateSponsors
```

Comment:
spelling, `Obfusticate` -> `Obfuscate`. This is in many other places too, but I think it's all the same misspelling, so find and replace should hopefully be e-z.